### DISCUSSION TITLE
Docker to compile Micropython image with IDF 5

### DISCUSSION THREAD

**Discussion by bixb922** [micropython/micropython#12619] [ESP32] [answered]
I have been using docker image (I believe by @mattytrentini, thanks a lot!!!) to compile Micropython. 

Are there updated images with IDF 5  available?

**Comment by andrewleech** [comment, marked as answer]
The official Espressif docker image works well for esp32 builds; `docker pull espressif/idf:v5.0.2`

**Comment by mattytrentini** [comment]
Yes, as @andrewleech mentioned, Espressif started publishing docker images so we no longer need to maintain our own. I keep my image online online so it doesn't break legacy builds. But I'm glad you were able to make use of it!
